addons/website/views/snippets/s_reviews_wall.xml

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_reviews_wall" name="Reviews Wall">
+    <section class="s_reviews_wall o_colored_level o_cc o_cc2 pt80 pb80">
+        <div class="container">
+            <div class="row align-items-stretch">
+                <div class="col-lg-12 pb32">
+                    <h2>Customers testimonials</h2>
+                    <p class="lead">What our customers are saying about us.</p>
+                </div>
+                <div data-name="Card" class="col-12 col-lg-4 pt16 pb16">
+                    <div class="s_card card o_cc o_cc1 h-100 my-0" data-vxml="001" data-snippet="s_card" data-name="Card" style="border-width: 0px !important;">
+                        <div class="card-body">
+                            <div class="s_rating pb16 s_rating_no_title" data-vcss="001" data-icon="custom" data-snippet="s_rating" data-name="Rating" data-active-custom-icon="fa fa-star text-warning" data-inactive-custom-icon="fa fa-star">
+                                <h4 class="s_rating_title o_default_snippet_text">Quality</h4>
+                                <div class="s_rating_icons o_not_editable">
+                                    <span class="s_rating_active_icons" style="color: var(--warning)">
+                                        <i class="fa fa-star"></i>
+                                        <i class="fa fa-star"></i>
+                                        <i class="fa fa-star"></i>
+                                        <i class="fa fa-star"></i>
+                                    </span>
+                                    <span class="s_rating_inactive_icons text-400">
+                                        <i class="fa fa-star"></i>
+                                    </span>
+                                </div>
+                            </div>
+                            <p class="card-text">
+                                "Collaborating with this team was seamless from start to finish. Their expertise and attention to detail made a huge difference for our project. I couldn’t be more satisfied with the outcome!"
+                            </p>
+                            <p class="card-text">
+                                <br/>
+                            </p>
+                            <h5 class="card-text">
+                                <img src="/web/image/website.s_company_team_image_1" class="s_reviews_wall_avatar img img-fluid me-auto float-start rounded-circle" alt=""/>
+                                Sullivan Mitchell
+                            </h5>
+                            <p class="card-text">
+                                <font class="text-600">Marketing Director</font>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div data-name="Card" class="col-12 col-lg-4 pt16 pb16">
```

Comment:
Hey @chgo-odoo, thanks for your work 👍 

Before reviewing this PR, I think it would be nice to use `s_blockquote` instead of `s_card` with custom content.
That way, we'd inherit blockquote's features and stay consistent with the design and features.
It would also be a good opportunity to improve `s_blockquote` by adding `s_rating` in this snippet with a toggle button to show/hide it 👍 

What do you think?